src/v3/src/components/PasswordRequirements/Icon.tsx

```diff
@@ -25,12 +25,13 @@ const Icon: FunctionComponent<PasswordRequirementIconProps> = (
   props,
 ) => {
   const { status } = props;
-  const statusToIconProps = {
-    incomplete: { name: 'close', color: 'grey.500' },
-    complete: { name: 'check-circle-filled', color: 'success' },
-    info: { name: 'information-circle', color: 'info' },
+  const statusToIcon = {
+    incomplete: { component: CloseIcon, color: 'grey.500' },
+    complete: { component: CheckCircleFilledIcon, color: 'success' },
+    info: { component: InformationCircleIcon, color: 'info' },
```

Comment:
Can we change the `color: .....` values to map to Odyssey 1.x tokens that we get from the `useOdysseyDesignTokens` hook? Also, right now I believe the `color: 'info'` does not exist in 1.x so the screenshots show this icon rendering as black/grey when it should actually map to `Tokens.HueBlue500` (I checked and 0.x used to map 'info' to blue 500 hue)

Reply:
Actually, I'd like @glenfannin-okta and maybe @alexdahl-okta 's opinions on this, but should we map the password requirement colors to all be palette-based rather than direct hues. I know we're aiming to give customers the ability to customize as much as possible and mapping to palette tokens rather than their underlying hues allows them to customize without breaking what they define "blue", "green", etc. to be.

If you agree that we should use palette values whenever possible, then I think these status colors should map to:
```
incomplete: { component: CloseIcon, color: 'Tokens.PaletteNeutralMain' }
complete: { component: CheckCircleFilledIcon, color: Tokens.PalettePrimarySuccess },
info: { component: InformationCircleIcon, color: Tokens.PalettePrimaryMain }
```

Reply:
Changed to proposed tokens

Reply:
Yeah, those more semantic tokens would be great! And those sound right – the main tokens for neutral/success/primary.

(Though are we still using the info status in password requirements? 🤔 May not matter here.)

Reply:
@alexdahl-okta We don't use info in password requirements.
However in case if server response contain new requirement that is not supported in SIW, it will render it with info icon:
<img width="288" alt="Screenshot 2023-12-01 at 11 43 57" src="https://github.com/okta/okta-signin-widget/assets/72614880/fe26251a-50df-4c98-b297-f6c1a2ef2fc2">